compiler/runtime/Trampoline.cpp

```diff
@@ -345,19 +345,7 @@ void arm64CodeCacheConfig(int32_t ccSizeInByte, int32_t *numTempTrampolines)
 
 void arm64CreateHelperTrampolines(void *trampPtr, int32_t numHelpers)
    {
-   uint32_t *buffer = (uint32_t *)((uint8_t *)trampPtr + TRAMPOLINE_SIZE);  // Skip the first trampoline for index 0
-
-   for (int32_t i=1; i<numHelpers; i++)
-      {
-      // LDR  PC, [PC, #-4]
-      // DCD  helperAddr
-      //
-      *buffer = 0xe51ff004;
-      buffer += 1;
-      *buffer = (intptrj_t)runtimeHelperValue((TR_RuntimeHelper)i);
-      buffer += 1;
-      }
-
+   TR_UNIMPLEMENTED();
```

Comment:
Why don't we just implement it properly instead?  Assuming we can use `x16` as one of the interprocedural scratch registers we can lay down a trampoline that looks like this:
```
   ldr x16, _helperAddress
   br x16
_helperAddress:
   .word <address of this helper>
```
The code for encoding for this (I believe) is:
```
      *buffer++ = 0x58000050;      // ldr x16, _helperAddress
      *buffer++ = 0xd61f0200;      // br x16
      *buffer++ = (intptrj_t)runtimeHelperValue((TR_RuntimeHelper)i);
```

Reply:
Changed the code based on the same function in the OpenJ9 project.